README.NL.rst

```diff
@@ -94,10 +94,10 @@ Quickstart
 
    .. code:: bash
 
-      $ wget https://raw.githubusercontent.com/maykinmedia/objects-api/master/docker-compose-quickstart.yml -O docker-compose.yml
-      $ docker-compose -f docker-compose-qs.yml up -d
-      $ docker-compose exec web src/manage.py loaddata demodata
-      $ docker-compose exec web src/manage.py createsuperuser
+      $ wget https://raw.githubusercontent.com/maykinmedia/objects-api/master/docker-compose.yml
+      $ docker compose up -d --no-build
+      $ docker compose exec web src/manage.py loaddata demodata
+      $ docker compose exec web src/manage.py createsuperuser
```

Comment:
Minor, but we could remove the dollar signs (also in the English readme) to make it easier to copy and paste

Reply:
'$' is removed